integration/client/container_idmapped_test.go

```diff
@@ -0,0 +1,124 @@
+//go:build linux
+// +build linux
```

Comment:
Not necessary if you append `_linux.go` to the file name

Reply:
Done